salt/modules/mount.py

```diff
@@ -960,3 +960,30 @@ def is_mounted(name):
         return True
     else:
         return False
+
+
+def get_othermountpoints(to_ignore=[]):
```

Comment:
We can't have mutable types as a default kwarg. Here's an explanation of why this is problematic: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments